modules/display.rb

```diff
@@ -0,0 +1,12 @@
+def list_options
+  puts '
+    Please choose one of the options: (1 - 7)
+      1. - List all books
+      2. - List all people
+      3. - Create a person
+      4. - Create a book
+      5. - Create a rental
```

Comment:
* Kindly fix this error i get when i try to create a rental 
![image](https://user-images.githubusercontent.com/52497246/200268845-5997afd4-345d-492b-9fb5-c49b075085cb.png)